doc/Programming-guide/Your-first-Kaa-application/index.md

```diff
@@ -422,14 +422,153 @@ int main(void)
 </div>
 
 <div id="app-java" class="tab-pane fade" markdown="1" >
+```java
+
+import org.kaaproject.kaa.client.DesktopKaaPlatformContext;
+import org.kaaproject.kaa.client.Kaa;
+import org.kaaproject.kaa.client.KaaClient;
+import org.kaaproject.kaa.client.SimpleKaaClientStateListener;
+import org.kaaproject.kaa.client.configuration.base.ConfigurationListener;
+import org.kaaproject.kaa.client.configuration.base.SimpleConfigurationStorage;
+import org.kaaproject.kaa.client.logging.strategies.RecordCountLogUploadStrategy;
+import org.kaaproject.kaa.schema.sample.Configuration;
+import org.kaaproject.kaa.schema.sample.DataCollection;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.Random;
+import java.util.Timer;
+import java.util.TimerTask;
+
+/**
+ * Class implement functionality for First Kaa application
+ *
+ * @author Maksym Liashenko
+ */
+public class FirstDemo {
+
+    private static final long DEFAULT_SAMPLE_TIME = 1000L;
```

Comment:
What is this about? Is it seconds? what is SAMPLE_TIME? I suspect that this is initial delay/period of the TemperatureTimerTask. This is confusing. 
It should be removed from the code. Once SDK is generated, it always contain default configuration. Put this value to the default configuration.